tests/installation/rescuesystem.pm

```diff
@@ -22,12 +22,8 @@ sub run {
     ensure_shim_import;
     $self->select_bootmenu_option('inst-rescuesystem', 1);
 
-    if (check_screen "keyboardmap-list", 100) {
-        send_key "ret";
-    }
-    else {
-        record_soft_failure;
```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2548

no point of having this, it never happend, at least what I can see from the previous tab